docs/uniwear-example.md

```diff
@@ -0,0 +1,140 @@
+
+This example provides an industrial use case of FL Suite for federated predictive maintenance. We assume you have already read and completed FL Suite's [quickstart](https://github.com/katulu-io/fl-suite/blob/beta/docs/quickstart.md).
+
+## Brief on tool-wear predictions 
+
+[Katulu](http://katulu.io) provides expertise and products in industrial AI; one of them is [custom tool-wear predictions across all machine tools](http://katulu.io/en/solutions/machine-tools/) within condition monitoring and predictive maintenance solutions. The challenge faced in the tool-wear use case is building models that can reliably predict the degradation of the machine tools and forecasting their Remaining Useful Life (RUL). 
+
+## Tool-wear dataset suitable for federated learning
+
+A federated model for custom tool-wear solutions should perform for multiple tool types and workpiece materials. Building robust models requires a good benchmark dataset with the multi-material tool and workpiece combinations. We prepare and merge two public milling datasets for easy use in federated machine learning tasks to fulfil this purpose. The dataset and associated 
+documentations  located in Katulu's [Uniwear dataset](https://github.com/katulu-io/uniwear-dataset) 
+repository.
+
+## A federated nonlinear regression model 
+
+* We provide an example use case for the FL suite in predictive maintenance leveraging the uniwear dataset.  
+* The model won't perform as Katulu's state-of-the-art models. We present the example as a realistic use case.
+* The model uses acquired mechanical signals directly without further feature extraction. One would need to extract meaningful features from the measured sensory signals.
+* Feature scaling applied only locally in clients for simplicity. We need FL Suite's federated analytics in computing global feature scaling as a preprocessing step.
+* Here, the time-series use case turned into a regression problem. This procedure would result in a significant loss of serial information. More advanced modelling requires state-of-the-art temporal deep learning models.
+* Our clients fetch the entire uniwear dataset and use a single signal as a subset. Each mechanical signal would be available to clients in separate streams in an actual deployment. 
+
+## Running the uniwear pipeline 
+
+The following statement imports the core package. 
+
+```python
+from fl_suite import pipelines
+```
+
+This package builds a federated learning pipeline. Preparing authentications to the compute cluster, client codes for distributing to clients, building compute nodes and starting the federated learning server. 
+ 
+Let's create a uniwear federated learning client. The client defines the model, loads the uniwear dataset from the repository address, shuffles the rows for regression, prepares test and training set, applies feature scaling, trains a nonlinear regression model, and reports Mean Absolute Percent Error (MAPE) performance per client. 
+
+```python
```

Comment:
nitpick: could you pass this python script through [black](https://github.com/psf/black) to follow the python styleguide we are following?